src/libs/ajax.js

```diff
@@ -833,19 +833,29 @@ const Workspaces = signal => ({
         return fetchOrchestration(`api/${root}/importEntities`, _.merge(authOpts(), { body: formData, signal, method: 'POST' }))
       },
 
-      importFlexibleEntitiesFile: file => {
+      importFlexibleEntitiesFile: async (isAsync, file) => {
```

Comment:
I don't love the name of the option `isAsync`. I judge this function is always async. I might call this option `shouldReadJsonBody` or `readBody` or somesuch.

Reply:
Ah, I see! The function itself is async, while the endpoint it's calling has a query parameter called `async`. So that could be confusing. What about `isAsyncImport` or `isAsyncJob`?

Reply:
The response from an async import looks like:
```
{   "url": "string",   "jobId": "string",   "workspace": {     "namespace": "string",     "name": "string"   } }
--

while the response from the synchronous import just returns the entity type as a string (so `participant` or `sample_set`) 



```

Reply:
I think this should be a separate entry in this file called `startAsyncFlexibleImport` or similar, then; generally, I want one definition here per endpoint.

Reply:
It is the same endpoint: https://firecloud-orchestration.dsde-dev.broadinstitute.org/#/Entities/flexibleImportEntities , whether or not it does the import asynchronously is decided by the `isAsync` query parameter

Reply:
or are you considering that to be different endpoints?

I have nothing against writing it as two separate functions - one for synchronous and one for async, especially since the responses are different. Just clarifying that, to me, this is a singular endpoint, and if we are following `one definition per endpoint` guideline, then it would be one definition

Reply:
Ahh, I missed that it was the same endpoint, since the returns are so different. I would treat it like a separate one for that reason.